base/applications/regedit/lang/ro-RO.rc

```diff
@@ -31,19 +31,19 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de registru.\n\
-  /D             Șterge o cheie de registru specificată.\n\
-  /E             Exportă conținutul unei chei de registru specificată într-un\n\
+  /C             Importă conținutul unui fișier de regiștri.\n\
```

Comment:
What is the benefit of all those "registru" -> "regiștri" changes? You do change an already translated term that is often used to one that contains a complex character. Which one of both does romanian MS Windows 2k3 use? Can you prove that with a picture?

Reply:
@JoachimHenze I think you are right. There is no benefit, so I will revert this change. Unfortunately, Windows XP/2k3 is not completely translated in Romanian, so the term registry is not translated anywhere. Hence, I cannot show you any proof. I will revert this change.

Reply:
resolved.